Glue Iceberg rest support (#8100)
fishtown-analytics/docs.getdbt.com: website/docs/docs/mesh/iceberg/snowflake-iceberg-support.md

```
Glue Data Catalog supports the Iceberg REST specification so that you can connect to Glue via the Iceberg REST API.

#### Table materialization in dbt-snowflake

> dbt-snowflake supports basic table materialization with a catalog-linked database in Glue using Iceberg REST. Note that incremental materializations are not yet supported.

Author: ```suggestion
Starting dbt Core v1.x, dbt-snowflake supports basic table materialization with a catalog-linked database in Glue using Iceberg REST. Note that incremental materializations are not yet supported.
```

> Starting dbt Core v1.x, dbt-snowflake supports basic table materialization with a catalog-linked database in Glue using Iceberg REST. Note that incremental materializations are not yet supported.

Author: ```suggestion
Starting dbt Core v1.x, dbt-snowflake supports basic table materialization on Iceberg tables registered in a Glue catalog through a catalog-linked database. Note that incremental materializations are not yet supported.
```

> Starting dbt Core v1.x, dbt-snowflake supports basic table materialization on Iceberg tables registered in a Glue catalog through a catalog-linked database. Note that incremental materializations are not yet supported.

Author: Could you please confirm what version this was introduced in?

Reviewer: it will be introduced in 1.11

> Starting dbt Core v1.x, dbt-snowflake supports basic table materialization on Iceberg tables registered in a Glue catalog through a catalog-linked database. Note that incremental materializations are not yet supported.

Author: ```suggestion
Starting with dbt Core v1.11, dbt-snowflake supports basic table materialization on Iceberg tables registered in a Glue catalog through a catalog-linked database. Note that incremental materializations are not yet supported.
```
